clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -50,6 +50,7 @@ public class GravitinoOptions {
   public static final String ROLE = "role";
   public static final String AUDIT = "audit";
   public static final String FORCE = "force";
+  public static final String COLUMNFILE = "csv";
```

Comment:
I used "csv" as it tells the user what format the file should be in, and it is less typing, but if you prefer "columnfile" or even "file" I can change that.

Reply:
My suggestion was that we may want to name an option by its purpose rather than its implementation (file format here). Consider that in the future, you may want to add another option where a CSV file happens to be a good fit. We will get trapped.